src/Schema/Directives/BaseDirective.php

```diff
@@ -54,17 +54,16 @@ abstract class BaseDirective implements Directive
      *
      * Lazily initialized.
      *
-     * @var array<string, mixed>
+     * @var array<string, mixed>|null
      */
-    protected array $directiveArgs;
+    protected ?array $directiveArgs = null;
```

Comment:
I would like to keep this as is, as setting this to `null` is not meant to be allowed.

Reply:
Changed to test an empty array and reset with an empty one instead of unset